bin/mksetup

```diff
@@ -7,7 +7,7 @@ try:
     eupsdir = sys.argv.pop(0)
     prefix = sys.argv.pop(0)
 except IndexError:
-    print >> sys.stderr, "Usage: mksetup EUPS_DIR EUPS_PATH [SETUP_ALIASES]"
+    sys.stderr.write("Usage: mksetup EUPS_DIR EUPS_PATH [SETUP_ALIASES]\n")
```

Comment:
if we are switching to Python 2.6+ syntax wouldn't we import `print_function` and use the `file=sys.stderr` scheme?